src/settingsui.ts

```diff
@@ -13,10 +15,13 @@ enum RefreshEnum {
 	reloadApp
 }
 interface IPaths { [path: string]: string }
+interface customUserscriptSettings { [key: string]: any }
```

Comment:
aren't the values here just JSON though?
because if yes, it can be described using something like this:
```ts
type JSONPrimitive = boolean | null | number | string;
type JSONValue = JSONPrimitive | JSONValue[] | {
    [key: string]: JSONValue;
};
type customUserscriptSettings = Record<string, JSONValue>
```

Reply:
I think using the already-made UserPrefs could just be better? Maybe define as an object whose keys are dynamic, which those keys have values that are just UserPrefs...
```js
type customUserscriptSettings = Record<string, UserPrefs>
```